addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14637,12 +14646,12 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#36210"
-msgid "Prevent the 'connection lost' notification window from displaying when the PVR backend server is not available."
+msgid "Prevent the \"connection lost\" notification window from displaying when the PVR backend server is not available."
```

Comment:
With your earlier change the setting itself uses `"Connection lost"` (capitalized) so you should probably do the same here for consistency.